### ISSUE TITLE
Sentence level context for kwic

### ISSUE THREAD

**Issue by cqx931** [dhowe/ritajs#195]
https://rednoise.org/rita/reference/RiTa/kwic/index.html
Currently kwic model only supports specifying the context through number of words.  There are ways to work around it and get the result within the current limit, but wouldn't it be helpful to also allow sentence level context in general?

Similar feature request:
https://github.com/quanteda/quanteda/issues/1189

**Comment by dhowe** [comment]
Yes, good suggestion. Did you have a method in mind for specifying the arguments ? Perhaps something like:

**Comment by cqx931** [comment]
@dhowe yes, other window types would be good!
